js/tests/Test_jsoo.ml

```diff
@@ -69,10 +72,8 @@ let _ =
          let argv =
            if filter <> "" then Array.append argv [| "-e"; filter |] else argv
          in
-         let tests =
-           [ Unit_parsing.tests (); Unit_engine.tests (); Unit_entropy.tests ]
```

Comment:
It looks like we lost these tests -- was that intentional? (or am I misreading the diff?)

Reply:
Nvm you fixed it